docs/en/setup/backend/backend-storage.md

```diff
@@ -280,6 +280,24 @@ storage:
 All connection-related settings, including URL link, username, and password are found in `application.yml`. 
 Only part of the settings are listed here. Please follow [HikariCP](https://github.com/brettwooldridge/HikariCP) connection pool document for full settings.
 
+## IoTDB
```

Comment:
I think you missed the option list on the top of file.

Reply:
Got it, I will fix it in the next commit